.github/ISSUE_TEMPLATE/update-external-links-gallery.md

```diff
@@ -5,40 +5,42 @@ title: ''
 labels: 'external-links-gallery-submission'
 ---
 
-<!-- Please fill out the template below. -->
+<!-- Please fill out the template below by adding the necessary information in between the single quotes in each relevant or known field. -->
 
 Make sure the following items are specified :
 
 - `url`
 - `title`
 - `description`
+- `authors` (please credit the creators of the resource in at least one method (`name`, `affiliation`, `affiliation_url`, or `email`)
+- `tag` (at least one tag is required, this helps users access the link by searching via the relevant `packages`, `formats`, or `domains`)
 
 **Note**:
 
 - The thumbnail can be a url to a direct image link.
```

Comment:
With the presence of the comment on line 25, should we remove this?

```suggestion
```

Reply:
Yes that is redundant. Thanks.